### PR TITLE
try bumping pip

### PR DESCRIPTION
**Pull request by LeeHughsVariant**
# Description

Orion needed pip bumped for some reason

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

```text
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
```

- [ ] Sanity Testing

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added documentation to test

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -39,7 +39,7 @@ RUN apt-get update &&\
   iptables \
   libdevmapper1.02.1 &&\
   rm -rf matching cache rm /var/lib/apt/lists/* &&\
-  pip install --upgrade --no-cache-dir wheel==0.37.1 pip==21.3.1
+  pip install --upgrade --no-cache-dir wheel==0.37.1 pip==22.0.4
```
Comment (vibindaniel) [inline Dockerfile:42]: Can this be done as part of the start script?

Comment (LeeHughsVariant) [reply Dockerfile:42]: Probably, but can we just get this in as a quick fix for now, and do that later?

Comment (LeeHughsVariant) [reply Dockerfile]: moved it to entrypoint script, but waiting for ravi to test.

Comment (LeeHughsVariant) [reply Dockerfile]: Ravi tested, this is working. Lmk if you have any other change requests.

File: action.yml [hunk 2]
```diff
@@ -21,6 +21,6 @@ inputs:
     required: true
 runs:
   using: "docker"
-  image: docker://public.ecr.aws/p4v7w0a5/lazy/python-action:v1.2.1
+  image: docker://public.ecr.aws/p4v7w0a5/lazy/python-action:test-latest
```
Comment (rchengvariant) [inline action.yml:24]: version bump

**Review by salyateem13 — APPROVED** [review]
Update image tag before merge

**Review by rchengvariant — APPROVED** [review]

**Pull request merged**